packages/@aws-cdk/aws-kinesis/test/test.stream.ts

```diff
@@ -109,15 +285,15 @@ export = {
 
     // THEN
     expect(stack).toMatch({
-      "Resources": {
-        "MyStream5C050E93": {
-          "Type": "AWS::Kinesis::Stream",
-          "Properties": {
-            "ShardCount": 1,
-            "RetentionPeriodHours": 24,
-            "StreamEncryption": {
-              "EncryptionType": "KMS",
-              "KeyId": "alias/aws/kinesis"
```

Comment:
I would just add

```
// tslint:disable:object-literal-key-quotes
```

to the top of the file. We do that in almost all tests, so that's it's easier to copy-paste from the template output.

Reply:
that linter rule does exist. My code formatter just did it's thing. I'll disable it next time :)